app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
```suggestion
  def new; end
```

method rỗng em viết như này nhé